source/Logging_main.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "./core/EasyLogging.hpp"
+
+
+using namespace clogged;
+
+void samplefunction() {
+    Logger::log << LOG_FNC << "This log message includes file and line number." << Logger::endl;
```

Comment:
I'm still uncomfortable with the `Logger::endl` being different from the `std::endl`, but it does look like both would work.  My inclination is to use the latter in the examples to make it easier for users.

I'm also iffy about requiring the `endl` at all, since users will forget to include it and be confused.  I propose an alternative below, but for now I think this will work okay.

(neither of the above suggestions require a change if you disagree, I'm just putting forth my perspective)

Reply:
A potential solution I was thinking for this is to reset the coloring, team name, and Debug level, when a different team is set

such as 

Logger::log << LOG_FNC << "This log message includes file and line number.";
Logger::log << TEAMEXAMPLE << "new team will reset the coloring, team name, and Debug level,";

Reply:
The problem there is that log messages might seem to be working, but appear at the wrong time, which might be confusing.  None-the-less, I think it will be okay as is.

Reply:
fixed they are equivalent now.